app/models/requests/request.rb

```diff
@@ -126,9 +126,8 @@ def route_requests(requestable_items)
     end
 
     def serial?
-      if doc[:format].present?
-        return true if doc[:format].include? 'Journal'
-      end
+      return unless doc[:format].present?
```

Comment:
These two lines might be better as  one line `doc[:format].present? && doc[:format].include? 'Journal'`

Reply:
thanks. I updated. .include? 'Journal' will return a rubocop lint error I think because of the parser version. I updated to .include?('Journal')